src/main/java/org/gridsuite/securityanalysis/server/WebFluxConfig.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.securityanalysis.server;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.powsybl.contingency.json.ContingencyJsonModule;
+import com.powsybl.security.json.SecurityAnalysisJsonModule;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.codec.ServerCodecConfigurer;
+import org.springframework.http.codec.json.Jackson2JsonDecoder;
+import org.springframework.http.codec.json.Jackson2JsonEncoder;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+import org.springframework.web.reactive.config.WebFluxConfigurer;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+@Configuration
+public class WebFluxConfig implements WebFluxConfigurer {
```

Comment:
where is this class used ?

Reply:
This is a configuration bean. So it is used to configure Spring framework, not to be accessed by our code.